lab_1_classify_by_unigrams/main.py

```diff
@@ -105,6 +189,29 @@ def detect_language(
 
     In case of corrupt input arguments, None is returned
     """
+    if unknown_profile is None or profile_1 is None or profile_2 is None:
+        return None
+    if (not isinstance(unknown_profile, dict) or not isinstance(profile_1, dict)
+            or not isinstance(profile_2, dict)):
+        return None
+
+    name1 = profile_1.get('name')
+    name2 = profile_2.get('name')
+    mse_1 = compare_profiles(unknown_profile, profile_1)
+    mse_2 = compare_profiles(unknown_profile, profile_2)
+    result1 = str(name1)
```

Comment:
Does this really need to be explicitly converted to a string?

Reply:
Actually, Mypy sees a mistake here in the type of variables if I remove str. I tried to fix this by doing explicit type check before result1 and result2. Not sure if it is what you expected me to do.